osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/periodicmeterreads/GetPeriodicMeterReadsCommandExecutorTest.java

```diff
@@ -101,7 +153,8 @@ public void testHappy() throws Exception {
         final DataObject bufferedObject = mock(DataObject.class);
         when(bufferedObject.getValue()).thenReturn(asList(data0, data1, data2, data3, data4, data5));
         final DataObject resultData = mock(DataObject.class);
-        when(resultData.getValue()).thenReturn(Collections.singletonList(bufferedObject));
+        when(resultData.getValue()).thenReturn(Arrays.asList(bufferedObject, bufferedObject));
```

Comment:
Je gebruikt 2 x bufferedObject. Mooier zou zijn om 2 verschillende bufferedObjects aan te maken. Het kost wat meer mock-werk, maar dan test je ook of de code bv niet per ongeluk altijd alleen maar de eerste waarde uit de lijst haalt.

Reply:
done, created one different mock, and asserted 2 values.